CHANGELOG.md

```diff
@@ -8,7 +8,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [2.32.0] - 2022-06-03
 
 ### Added
-
+- Add support for hosting meetings in US GovCloud regions.
```

Comment:
Technically we don't require `CHANGELOG.md` entries for demo changes (it will only check `src`).

Reply:
ok, I'll remove.